### DISCUSSION TITLE
Liquid logic operator question.

### DISCUSSION THREAD

**Discussion by gasatrya** [11ty/eleventy#2547] [Q&A]
I just switch to liquid from nunjucks, I read the documentation, Liquid is also support logic operator but the result is different.

`{{ description or site.description }}`

Nunjucks will show `description` if not empty, if empty will just show `site.description`, but in Liquid, it just show `true`. Is this how liquid work?

**Comment by pdehaan** [comment]
I believe in LiquidJS you can use the [`default`](https://liquidjs.com/filters/default.html) filter.

```njk
---
title: Custom title
---

title={{ title | default: site.title }}
description={{ description | default: site.description }}
```
